### ISSUE TITLE
Add "relative" behaviour to Timestamp container

### ISSUE THREAD

**Issue by bcmn** [bbc/simorgh#1279]
**Is your feature request related to a problem? Please describe.**
UX have updated the designs for the Timestamp components.

Blocked on [Psammead#277](https://github.com/bbc/psammead/issues/277).

**Describe the solution you'd like**
Add the "relative" logic to the Timestamp container. This issue will not yet integrate the relative timestamp onto the page.

We suggest adding a file within the TimestampContainer that handles inputs of datetime and outputs the relative string. This PR should not handle including these relative strings in the Timestamp Container. 

**Logic:**
The text output should vary according to the following rules:

| received timestamp is... | rendered text |
|-------------|-----------|
| < 2 min ago | 1 minute ago |
| < 60 min (1 hour) ago | X minutes ago |
| < 120 min (2 hours) ago | 1 hour ago |
| > 120 min (2 hours) ago | X hours ago |

NB, beyond 10 hours we will switch to absolute timestamps. This is covered in #1280.

Potential edge case of GMT/BST change.
“X hours ago…” should always be the “real” value (if a user started a stopwatch & let it run from the moment the article was published), should not change with the +/- hour.

**Describe alternatives you've considered**

**Testing notes**
Happy for the unit tests to cover this part of the timestamp implementation. It would be good to go through the unit test logic once the dev has completed them.

Dev insight: We should have unit tests covering each of the cases in the relative table. Note: the actual timestamp component will not reflect the relative work yet. That will be carried out in a future issue. https://github.com/bbc/simorgh/issues/1280

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

**Comment by bcmn** [comment]
Comment from @greenc05  regarding `<time>` element:

> I’m not sure semantically that putting the word ‘Updated’ into a time element is correct? What do we think? If it’s not correct, this would mean the markup would be something like:
> 
> `<span class=“timestamp__updated”>Updated <time datetime=“2018-05-26">26 May 2018</time></span>`
> 
> This will cause text splitting in VoiceOver iOS e.g. one swipe for updated, and another swipe for ‘26 May 2018’, and these swipes are in addition to the swipe already done for the published time, so that would be 3 swipes to get passed the timestamp in this case… not that efficient…
> 
> To make this more efficient… We should consider if it would be correct/ok to put the word ‘Updated’ inside the time element? For example:
> 
> `<time class=“timestamp__published” datetime=“2018-05-26"> Updated 26 May 2018</time>`
> 
> If it’s not correct/ok to do that, is it worthwhile using the time HTML element, what benefits does this bring us other than correct semantics? Could we drop the time element to improve the screen reader UX here? For example:
> 
> `<span class=“timestamp__published” datetime=“2018-05-26"> Updated 26 May 2018</span>`
> 
> And if we decided to drop the time element for the updated time stamp, should we also not use the time tag for the published time stamp (for consistency)? Note the only screen reader I have seen the time element semantics conveyed is with Talkback.

As mentioned above, adding phrases like "updated" to the `<time>` element seems to be going against the W3 examples for it.

I think we should be clear about what benefit adding the `<time>` element gives us, who/what uses or benefits from it, & whether it's worth this trade-off for AT users.

Furthermore, it's important to remember we favour `<span>` tags for stuff like this:

> For the markup - Regardless of the time element can we use nested span’s rather than div’s please.
> 
> For example:
> ```
> <div class=“timestamp”>
>        <span>27 May 2018</span>
>        <span>Updated 4 minutes ago</span>
> </div>
> ```
> 
> Rather than:
> ```
> <div class=“timestamp”>
>        <div>27 May 2018</div>
>        <div>Updated 4 minutes ago</div>
> </div>
> ```
> 
> The span elements used are better than div’s, as both the timestamps are read out together with one press of the down arrow with Jaws, where as when div’s are used instead of the span elements, 2 presses of the down arrow are needed - presumably because a div is a block element
